src/sage/rings/finite_rings/conway_polynomials.py

```diff
@@ -127,6 +133,7 @@ class PseudoConwayLattice(WithEqualityById, SageObject):
 
     EXAMPLES::
 
+        sage: # needs sage.rings.finite_rings
         sage: from sage.rings.finite_rings.conway_polynomials import PseudoConwayLattice
```

Comment:
Isn't this file installed by the distribution that provides the feature?

Reply:
No, see below

Reply:
OK.